src/cozmo/robot.py

```diff
@@ -931,6 +931,26 @@ def play_anim_trigger(self, trigger, loop_count=1):
         self._action_dispatcher._send_single_action(action)
         return action
 
+    def set_idle_animation(self, anim_trigger):
+        '''Set the Idle Animation on Cozmo
```

Comment:
i think some additional documentation on what "idle" means would be useful here... does it automatically restart the animation if it goes un-idle and then back again? etc.